admin/extend2da.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!python3
```

Comment:
This is not portable enough, please use a patch downstream.
I guess `/usr/bin/env python` is also not sufficient on Haiku?

Reply:
Ah sorry, I botched that first commit, I meant to write it as: #!/usr/bin/python3
I fixed it in the second commit I think, it should be fine now :)

Btw Haiku accepts both #!python3 and #!/usr/bin/python3 so either syntax is fine for the systen

Reply:
Ah ok, I'll revert that change then

Reply:
For the cache, we only check that the dir is empty, so you don't lose any files. Beyond that standard permission limits apply.